### ISSUE TITLE
Failed to copy resources: index.html and myproperties.properties in multimodule projects

### ISSUE THREAD

**Issue by Rashmini** [quarkusio/quarkus#4697]
**Describe the bug**
After compiling the application and trying to run, it prompts the error: **Failed to copy resources: java.nio.file.NoSuchFileException: C:\Users\rnarlk\projects\quarkus-multimodule-issue\rest\target\classes\myproperties.properties**

**Expected behavior**
Strings in myproperties.properties file should be displayed in http://localhost:8080/welcome

**Actual behavior**
Here is the stacktrace
```
2019-10-21 15:46:36,550 ERROR [io.qua.dev] (vert.x-worker-thread-1) Failed to copy resources: java.nio.file.NoSuchFileException: C:\Users\rnarlk\projects\quarkus-multimodule-issue\rest\target\classes\myproperties.properties
        at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:79)
        at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:97)
        at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:102)
        at sun.nio.fs.WindowsFileSystemProvider.implDelete(WindowsFileSystemProvider.java:269)
        at sun.nio.fs.AbstractFileSystemProvider.delete(AbstractFileSystemProvider.java:103)
        at java.nio.file.Files.delete(Files.java:1126)
        at io.quarkus.dev.RuntimeUpdatesProcessor.checkForFileChange(RuntimeUpdatesProcessor.java:332)
        at io.quarkus.dev.RuntimeUpdatesProcessor.doScan(RuntimeUpdatesProcessor.java:121)
        at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:52)
        at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:44)
        at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$2(ContextImpl.java:316)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.lang.Thread.run(Thread.java:748)

2019-10-21 15:46:36,564 ERROR [io.qua.dev] (vert.x-worker-thread-1) Failed to copy resources: java.nio.file.NoSuchFileException: C:\Users\rnarlk\projects\quarkus-multimodule-issue\html\target\classes\META-INF\resources\index.html
        at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:79)
        at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:97)
        at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:102)
        at sun.nio.fs.WindowsFileSystemProvider.implDelete(WindowsFileSystemProvider.java:269)
        at sun.nio.fs.AbstractFileSystemProvider.delete(AbstractFileSystemProvider.java:103)
        at java.nio.file.Files.delete(Files.java:1126)
        at io.quarkus.dev.RuntimeUpdatesProcessor.checkForFileChange(RuntimeUpdatesProcessor.java:332)
        at io.quarkus.dev.RuntimeUpdatesProcessor.doScan(RuntimeUpdatesProcessor.java:121)
        at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:52)
        at io.quarkus.vertx.http.deployment.devmode.VertxHotReplacementSetup$1.handle(VertxHotReplacementSetup.java:44)
        at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$2(ContextImpl.java:316)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.lang.Thread.run(Thread.java:748)
```

**To Reproduce**
1. Clone the repository from https://github.com/Rashmini/Quarkus-multimodule-project-sample.git
2. Compile the project using the command _mvn compile quarkus:dev_
3. Go to http://localhost:8080/welcome

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: Microsoft Windows [Version 10.0.17763.737]
 - Output of `java -version`: 1.8.0_221
 - Quarkus version or git rev: 0.25.0

**Additional context**
This works fine with Quarkus 0.23.2 and only happens in multimodule projects.

**Comment by jaikiran** [comment]
Perhaps this is related to this https://github.com/quarkusio/quarkus/pull/4633 ?

**Comment by Rashmini** [comment]
This was solved by release 0.26.0.
Thank you
